### PR TITLE
Refactoring, Unit Tests, FastAPI integration

### PR DESCRIPTION
**Pull request by GetLouied**
- Refactored code to make it more legible and organized. 
- Created a unit test for popping the time out of the json file and saving the unix time
- Began integrating FastAPI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+todo.txt
+**/__pycache__/
```
Comment (RobertLD) [inline .gitignore]: Change to `**/__pycache__/**`

File: GETest
Comment (RobertLD) [inline GETest:1]: Add **/*.db to the gitignore

**Review by RobertLD — CHANGES_REQUESTED** [review]
This will be a lot easier to review when you fix the .gitignore issues

**Comment by GetLouied** [comment]
Okay, I modified the git ignore file

**Comment by RobertLD** [comment]
@GetLouied  try `git reset --soft HEAD~2` and `git commit` and `git push -f` to remove the bad files

**Comment by GetLouied** [comment]
that automatically closed the PR

**Pull request closed without merging**